test/integration/node/util.go

```diff
@@ -0,0 +1,350 @@
+package node
```

Comment:
Perhaps I'm missing something, but where is this `util.go` code coming from? I don't see the equivalent code removed anywhere. Are we duplicating it?

Reply:
Actually, some functions which are used by the integration test of TaintBasedEvictions exists in the `test/integration/scheduling/util.go`. In my opinion, if we move the TaintBasedEviction test to the sig-node, we shouldn't import unnecessary functions from sig-scheduling.

Reply:
Hmm, I definitely agree that we don't want to import from `test/integration/scheduling/util.go`. However, I'd prefer to avoid the duplication if we can. Can we move `util.go` to a more general package which both `scheduling` and `node` could safetly import from?

Alternatively, if there's a strict timeline for getting this merged, we could also address this duplication in a later pr. If we go that route, could you please add a TODO to resolve the duplication?

Reply:
`test/integration/util` would be a good place.

Reply:
Yep. Let me move it .